client/app/controllers/collection/add.js

```diff
@@ -258,36 +258,36 @@ export default Ember.Controller.extend({
     },
 
 
-    saveParameter_signature: ['parameter', 'updated_parameter'],
-    saveParameter: function(parameter, updated_parameter, parameters) {
-        if (typeof updated_parameter.value !== 'undefined') {
-            Ember.set(parameter, 'value', updated_parameter.value);
+    saveParameterSignature: ['parameter', 'updatedParameter'],
+    saveParameter: function(parameter, updatedParameter, parameters) {
+        if (typeof updatedParameter.value !== 'undefined') {
+            Ember.set(parameter, 'value', updatedParameter.value);
         }
-        if (typeof updated_parameter.state !== 'undefined') {
-            Ember.set(parameter, 'state', updated_parameter.state);
+        if (typeof updatedParameter.state !== 'undefined') {
+            Ember.set(parameter, 'state', updatedParameter.state);
         }
         Ember.run.next(this, function() {
             this.get('updateState').call(this, this.get('formActions'));
         });
     },
 
 
-    closeSection_signature: ['sectionName'],
+    closeSectionSignature: ['sectionName'],
     closeSection: function(sectionName, parameters) {
         this.get('panelActions').close(this.get(`_names.${this.get('_names').indexOf(sectionName)}`));
```

Comment:
in ```_names``` don't use trailing or leading underscores
https://github.com/airbnb/javascript#naming--leading-underscore